### DISCUSSION TITLE
Is it possible to set a just a proxy to the wild internet for some `VIRTUAL_PATH`?

### DISCUSSION THREAD

**Discussion by assapir** [nginx-proxy/nginx-proxy#2233] [Q&A] [answered]
We have several endpoints, which go to some service, based on their path, like `VIRTUAL_PATH: ~* /update/*` etc.
We want to have a `VIRTUAL_PATH` (let's say `~* /boop/*`) to directly proxy to `https://boo.boop.com/boop`, without having a docker service for it. is that possible?

**Comment by buchdag** [comment]
Hi. Could you try re phrasing the question with maybe more examples ? I don't really get what behaviour you're trying to achieve.

**Reply by assapir** [reply]
Sorry for being unclear. 
Let's say we have 3 services that I want to use nginx-proxy for, all can run locally, where the routing is based on `VIRTUAL_PATH` : `/a` going to service 1, `/b` going to service 2 and so on. 
Service 26, with routing `/z` can't run locally, but instead it is hosted online. 
Can I have some how route all request to `/z` going directly to the internet url (`example.com/z/`) without defining a 4th service which is only nginx doing some proxy?

Sorry again for not being clear.

**Reply by buchdag** [reply, marked as answer]
Oh ok I get it. Unfortunately no we don't support proxying to something else than a container.

It might be added one day but that's low priority, has this is outside the (original) stated project scope:

> Automated nginx proxy **for Docker containers** using docker-gen

**Reply by assapir** [reply]
Thanks! It might be easy to implement by adding an env var that can be any Nginx location block.
